templates/import_form.twig

```diff
@@ -69,6 +69,33 @@ jQuery(document).ready(function($) {
         }
     }
 
+    // Enables the confirm dialog when labels have been modified 
+    $('#labels-form').on('change', function(){
+        window.onbeforeunload = confirmOnPageExit;
+    });
+
+    // Disables the confirm dialog when saving the changes
+    $('#save-button, #save-button-bottom').click(function(){
+        window.onbeforeunload = null;
+    }); 
+
+    var confirmOnPageExit = function (e) 
+    {
+    // If we haven't been passed the event get the window.event
+    e = e || window.event;
+
+    var message = 'Any text will block the navigation and display a prompt';
```

Comment:
Yes right. I thought it did not matter the message being used because all different browsers I used to test were not returning that message. I don't know if the returned message will ever be displayed but in the case it does in other browser versions maybe we should change it for `This page is asking you to confirm that you want to leave - data you have entered may not be saved.` Let me know and I will create a new commit for it.